src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriterILGen.cs

```diff
@@ -14,7 +14,13 @@
 using System.Xml.Extensions;
 
 #if !uapaot
+#if XMLSERIALIZERGENERATOR
+using System.Xml;
+
+namespace Microsoft.XmlSerializer.Generator
+#else
 namespace System.Xml.Serialization
+#endif
 {
```

Comment:
This file is not included in Sgen anymore. Can you please revert the changes in this file?

The same for ReaderILGen, ILGen ..